projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/path-actions/class-filter-older.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Path_Actions;
+
+use SplFileInfo;
+
+class Filter_Older implements Path_Action {
+	private $timestamp;
+	private $sub_action;
```

Comment:
Curious: Why is this `$sub_action` instead of `$action`?

Reply:
Adnan wrote this, but maybe because this filter is an action itself, so $sub_action clearly separates the action to perform from the filter.